app/services/api/charts/company.rb

```diff
@@ -0,0 +1,121 @@
+module Api
+  module Charts
+    class Company
+      attr_reader :company
+
+      SECTORS_LEVELS_DESC = {
+        '0' => 'Unaware of Climate Change as a Business Issue',
+        '1' => 'Acknowledging Climate Change as a Business Issue',
+        '2' => 'Building Capacity',
+        '3' => 'Integrating into Operational Decision Making',
+        '4' => 'Strategic Assessment'
+      }.freeze
+
+      def initialize(company)
+        @company = company
+      end
+
+      def details_data
+        {
+          name: company.name,
```

Comment:
missing the @ before `company`?

Reply:
I've added attr_accessor for this, but maybe using `@ivar` is better here.

Also, I'll move this `details_data` to another class (`Api::Presenter::Company` ?), as this is not related to charts at all.

Reply:
it might be my rusty rails 😅 but because you setup `@company` on the initializer I thought you had to use that. Like you were doing in the sector.rb file with `@company_scope`

Reply:
Yeah, maybe presenter would be a good place for that.

Reply:
f7b5f29